contents/julia_basics.md

```diff
@@ -451,7 +451,7 @@ scob(s)
 A lot of times, we don't care about the name of the function and want to quickly make one.
 What we need are **anonymous functions**.
 They are used a lot in Julia's data science workflow.
-For example, when using `DataFrames.jl` (@sec:dataframes) or `Plots.jl` (@sec:datavisPlots), sometimes we need a temporary function to filter data or format plot labels.
+For example, when using `DataFrames.jl` (@sec:dataframes) or `Plots.jl` (@sec:DataVisualizationPlots), sometimes we need a temporary function to filter data or format plot labels.
```

Comment:
```suggestion
For example, when using `DataFrames.jl` (@sec:dataframes) or `Plots.jl` (@sec:plots), sometimes we need a temporary function to filter data or format plot labels.
```
The section label "plots" would have been enough, I think. The whole point of a section label is that it may differ from the section name (and can be more stable).

But, this comment is nitpicking. It doesn't really matter.

Reply:
If we accept @rikhuijzer's suggestion, we need to also change in the other files such as `preface.md'

Reply:
Indeed, for now I did changed it everywhere, so it should work. If @rikhuijzer commits the suggestions in all other places I'm fine with it. :D